### PR TITLE
Added Python Solution for cses-2428(Subarray Distinct Values)

### PR DESCRIPTION
**Pull request by gsvamsi18**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-2428.mdx [hunk 1]
```diff
@@ -74,4 +74,42 @@ public class cses2428 {
...
+ans = 0
+distinct = 0
+freq = {}
+while left < n :
+    while right < n :
+        if arr[right] not in freq and distinct == k:
+            break
+
+        # Put the new array element in the map.
+        if arr[right] in freq :
```
Comment (envyaims) [inline solutions/cses-2428.mdx:94]: keep your spacings consistent
```suggestion
        if arr[right] in freq:
```

File: solutions/cses-2428.mdx [hunk 2]
```diff
@@ -74,4 +74,42 @@ public class cses2428 {
...
+            freq[arr[right]] += 1
+        else:
+            freq[arr[right]] = 1
+            distinct += 1
+        right += 1
+
+    # Add the new subarrays.
+    ans += right - left
+    freq[arr[left]] -= 1
```
Comment (envyaims) [inline solutions/cses-2428.mdx:103]: move this to the bottom so the comment makes sense

**Review by envyaims — CHANGES_REQUESTED** [review]

**Comment by envyaims** [comment]
also use tabs

File: solutions/cses-2428.mdx [hunk 3]
```diff
@@ -74,4 +74,42 @@ public class cses2428 {
 
 </JavaSection>
 
+<PySection>
+
+```py 
+
```
Comment (SansPapyrus683) [inline solutions/cses-2428.mdx:80]:
```suggestion
```

**Review by ryanchou-dev — CHANGES_REQUESTED** [review]
space out ops and use tabs

File: solutions/cses-2428.mdx (hunk 1 above)
Comment (gsvamsi18) [reply solutions/cses-2428.mdx]: updated.

File: solutions/cses-2428.mdx (hunk 2 above)
Comment (gsvamsi18) [reply solutions/cses-2428.mdx]: done

File: solutions/cses-2428.mdx (hunk 3 above)
Comment (gsvamsi18) [reply solutions/cses-2428.mdx]: done.

**Comment by gsvamsi18** [comment]
Anything else to update

File: solutions/cses-2428.mdx [hunk 4]
```diff
@@ -74,4 +74,39 @@ public class cses2428 {
 
 </JavaSection>
 
-</LanguageSection>
+<PySection>
+
+```py 
+n,k = map(int,input().split())
```
Comment (SansPapyrus683) [inline solutions/cses-2428.mdx:80]: space out operators

Comment (gsvamsi18) [reply solutions/cses-2428.mdx:80]: n, k = map( int, input().split() )

Is this how you want the spaces if not please tell me.

Comment (SansPapyrus683) [reply solutions/cses-2428.mdx:80]: uh like
```suggestion
n, k = map(int, input().split())
```

**Comment by gsvamsi18** [comment]
any more changes ?

File: solutions/cses-2428.mdx (hunk 4 above)
Comment (gsvamsi18) [reply solutions/cses-2428.mdx]: updated

**Comment by SansPapyrus683** [comment]
hold on after tests complete

**Review by SansPapyrus683 — APPROVED** [review]

**Review by envyaims — APPROVED** [review]
kay

**Pull request merged**